SeaPublicWebsite/Views/EnergyEfficiency/OtherHeatingType.cshtml

```diff
@@ -26,15 +26,16 @@
             
             @{
                 Func<object, object> blueBannerHint = null;
-                if (Model.Epc?.OtherHeatingType is not null)
+                if (Model.Epc?.EpcHeatingType is EpcHeatingType.Biomass or EpcHeatingType.CoalOrSolidFuel or EpcHeatingType.Other)
```

Comment:
Assuming we surface the epc recommends a user has a gas boiler, but they still proceed to 'Other heating type', then I suppose we assume they know better and not show anything here...?